applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -129,10 +126,14 @@ static void le_audio_event_publish(enum le_audio_evt_type event, struct bt_conn
 	ERR_CHK(ret);
 }
 
+K_MUTEX_DEFINE(mtx_unicast_group_created);
```

Comment:
Looks a bit strange that we have a mutex here and a bool for "the same" on line 95.

Reply:
The bool in line 95 is for making sure the unicast group is created once.
The mutex in here is make sure only one "group create" can happen at the same time.

Reply:
Ah, I see. Then perhaps we can rename the first to `unicast_group_created_once` ? I would like to discuss this, perhaps together with Alexander.